core/tests/karma-globals.js

```diff
@@ -17,7 +17,21 @@
  */
 
 var GLOBALS = {
-  INTERACTION_SPECS: {},
+  INTERACTION_SPECS: {
+    TextInput: {
+      customization_arg_specs: [
+        {},
```

Comment:
You could JSON.stringify() and then console.log() it perhaps -- then unparse in python. To clarify, though, what I meant was to just replicate the TextInput bit.